openpype/hosts/traypublisher/plugins/publish/collect_editorial_reviewable.py

```diff
@@ -0,0 +1,24 @@
+import pyblish.api
+
+
+class CollectEditorialReviewable(pyblish.api.InstancePlugin):
+    """Collect reviwiewable toggle to instance and representation data
+    """
+
+    label = "Collect Editorial Reviewable"
+    order = pyblish.api.CollectorOrder
+
+    families = ["plate", "review", "audio"]
+    hosts = ["traypublisher"]
+
+    def process(self, instance):
+        creator_identifier = instance.data["creator_identifier"]
+        if "editorial" not in creator_identifier:
```

Comment:
The explicit checks should be at all places, especially in tray publisher where creators are "free for all". Please change it here and in `CollectShotInstance`.